src/Engine/EmitMSIL/CodeGenIL.cs

```diff
@@ -1064,10 +1064,12 @@ private void EmitArray<T>(Type arrType, IEnumerable<T> items, Action<T, int> ite
         private void EmitIndexingForIList(AssociativeNode array, AssociativeNode index, Type collectionType, Type listElementType)
         {
             var indexT = DfsTraverse(index);
-            var mi = typeof(IList).GetMethod("get_Item", BindingFlags.Instance | BindingFlags.Public);
```

Comment:
Did `GetMethod` not work? I used `GetProperties` but I was curious if you could get it to work using `GetMethod`.

Reply:
Ah, ok.